platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/GuestClaimsHandlerExt.java

```diff
@@ -101,8 +106,8 @@ public GuestClaimsHandlerExt(
   /** Called by the container to initialize the object. */
   public void init() {
     try (InputStream inputStream = new FileInputStream(profilesFilePath)) {
-      String json = IOUtils.toString(inputStream);
-      this.profiles = (Map<String, Object>) Boon.fromJson(json);
+      String json = IOUtils.toString(inputStream, StandardCharsets.UTF_8);
```

Comment:
✏️ We were forced to read it as a string first with Boon. Should we take advantage of Gson to parse a stream directly?